sycl/test/basic_tests/SYCL-2020-spec-constants.cpp

```diff
@@ -0,0 +1,98 @@
+// RUN: %clangxx -fsycl -fsycl-device-only -c -o %t.bc %s
+// RUN: sycl-post-link %t.bc -spec-const=rt -o %t-split1.txt
+// RUN: cat %t-split1_0.prop | FileCheck %s
+// RUN: sycl-post-link %t.bc -spec-const=default -o %t-split2.txt
+// RUN: cat %t-split2_0.prop | FileCheck %s
+// RUN: llvm-spirv -o %t-split1_0.spv -spirv-max-version=1.1 -spirv-ext=+all %t-split1_0.bc
```

Comment:
It looks like it is better to move the test to clang tests and implement some stub for SYCL RT classes to let it build.

Reply:
Good point, will move it there

Reply:
After thinking about this a bit more, I'm not sure that it will work there: `clang` doesn't have dependency on `sycl-post-link` and `llvm-spirv`, so it would be a significant change there, while SYCL LIT tests already depend on `sycl-toolchain`, which includes both tools.

Also, this seems like an integration test and I'm not sure that we have a better place for it. Note: it is not a complete E2E test, so I'm not sure that `llvm-test-suite` is a proper place for it either